decentraland/unity-renderer
# Feat: Loading Avatar Hologram

## What does this PR change?



This PR adds the visual elements required to make a transition from an avatar that is loading to a full loaded one. The base shader used by all the avatars has been modified to support the transition effect, and new shaders and particle effects have been added.

## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer-branch={branch_name}&{desired_url_params}
2. Verify the avatars in the game are render properly.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
